.github/workflows/build.yml

```diff
@@ -36,5 +36,5 @@ jobs:
       run: ./mvnw -B formatter:validate install --file pom.xml
 
     - name: Build - Native
-      run: ./mvnw -B verify -Pnative -Dquarkus.native.container-build=true --file integration-tests/pom.xml
+      run: cd integration-tests && ../mvnw -B verify -Pnative -Dquarkus.native.container-build=true
```

Comment:
Nitpick: Why not `-f  integration-tests`?

Reply:
`cd` tends to be safer for some buggy plugins (e.g. assuming that `Paths.get(".")` always returns `${basedir}`). Maybe we do not have any such here.

Reply:
We probably don't but in the end I don't mind `cd`.